examples/stress_tests/many_cubes.rs

```diff
@@ -1,8 +1,21 @@
+//! Simple benchmark to test per-entity draw overhead
+//!
+//! To a realistic performance result, run this in release mode.
```

Comment:
It seems like there's a missing word here. "To see a" or "To get a" or something like that.

Reply:
```suggestion
//! To measure performance realistically, be sure to run this in release mode.
```